packages/opentelemetry-types/src/metrics/Metric.ts

```diff
@@ -0,0 +1,75 @@
+/**
+ * Copyright 2019, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Resource } from '../resources/Resource';
+
+/**
+ * Options needed for metric creation
+ */
+export interface MetricOptions {
+  /** The description of the Metric. */
+  description?: string;
+
+  /** The unit of the Metric values. */
+  unit?: string;
```

Comment:
Should we specify the implied default unit if this is not specified? E.g. "count"? And should we specify canonical units for things like bytes, seconds, bytes/second, count/second, etc.?

Is there a formal spec for this? Should we adopt something like http://metrics20.org/spec/#tag-values-unit ?

Reply:
Added the default value, based on Java's implementation: https://github.com/open-telemetry/opentelemetry-java/blob/master/api/src/main/java/io/opentelemetry/metrics/Metric.java#L93